src/client/app/menu.react.js

```diff
@@ -8,15 +10,15 @@ class Menu extends Component {
     return (
       <header>
         <h1>
-          <a href="https://github.com/steida/este">Este.js</a> App
+          <FormattedHTMLMessage message={msg('menu.headerHtml')} />
```

Comment:
What do you think about using Html suffix for messages with HTML? This way nobody forgets to use FormattedHTMLMessage. menu.header.linkToEste sound much better for the link itself, but I'm not so sure about message which just contains linkToEste. Another thing is, that in the past when I start to fiddle with some new app, the first thing I do, is to put some nice custom header (name of my new application). That's why I used somehow 'general' naming for texts, which I usually replace as first. But I gladly change it, if you see some reason for having explicitly named placeholder in this case.